### DISCUSSION TITLE
Akses Lokal

### DISCUSSION THREAD

**Discussion by shobalubis19** [ikhsan3adi/absensi-sekolah-qr-code#132] [Q&A]
Permisi mau tanya, Kalok untuk bisa diakses dengan jaringan yang sama bagaimana ya?

**Comment by reactmore** [comment]
> Permisi mau tanya, Kalok untuk bisa diakses dengan jaringan yang sama bagaimana ya?

Tinggal panggil aja IP : port nya atau kalo pake custom virtualhost tinggal panggil server alias nya

**Comment by shobalubis19** [comment]
Permisi kak mau tanya lagi nih. Maaf kalok ngerepotin.

Ini kan prejectnya saya buka dengan docker kan. Tapi ini buat access kameranya ke blokir. Padahal kemarin saya pakek xampp bisa.
Ada solusinya ga?
